modules/apps/transfer/keeper/relay_test.go

```diff
@@ -337,6 +337,25 @@ func (suite *KeeperTestSuite) TestOnRecvPacket() {
 				expEscrowAmount = sdkmath.NewInt(100)
 			}, true, false,
 		},
+		{
+			"failure: receive is disabled",
+			func() {
+				suite.chainB.GetSimApp().TransferKeeper.SetParams(suite.chainB.GetContext(),
+					types.Params{
+						ReceiveEnabled: false,
+					})
+			}, false, false,
+		},
+		{
+			"failure: receive is disabled - chainB is source",
```

Comment:
I'm not familiar enough with the codebase to know if this case is needed. Happy to delete it in case it is not :)

Reply:
I think it's not strictly necessary but should do no harm in including it!

Reply:
I think I would remove just to keep it the test tight and clean, but will not fight over it. :)

Reply:
I removed it :) I was a bit on the verge, not being familiar enough with the codebase.